apps/examples/csifw_test/csifw_test.c

```diff
@@ -0,0 +1,205 @@
+#include "csifw_test.h"
+#include "csifw/csifw_api.h"
+
+#include <stdio.h>
+
+
+/****************************************************************************
+ * csifw_test
+ ****************************************************************************/
+
+#include <fcntl.h>
+static int gFd; /* File descriptor*/
+
+#define PRNT_LN1 printf("______________________________________________________________________________________________________\n")
+#define PRNT_LN2 printf("______________________________________________________________________________________________________\n\n")
+#define PRNT_LN3 printf("\n\n______________________________________________________________________________________________________\n")
+
+
+static unsigned int gCounter;
+
+static int print_log(const char *format, ...)
+{
+	
+	va_list args;
+	va_start(args, format);
+
+	vprintf(format, args);
+	va_end(args);
+	return 1;
+}
```

Comment:
Could you let me know why you use this function instead of `printf`? Any advantage???

Reply:
Changed to printf.